### DISCUSSION TITLE
Feature Request:  Add ability to import and export the content of multiline input.text_area() settings.

### DISCUSSION THREAD

**Discussion by scheerd** [akumidv/tradingview-assistant-chrome-extension#177] [Ideas]
This is one of 2 things that would be nice to have handled in export/import.
Currently input.text_area() settings are ignored, and input.source() settings are ignored.  Both have string equivalents that should be handled by TV.  input.text_area() exports from TV Properties Export as multiline in double-quotes, which is recognized by many csv parser libraries as a single line with return characters.

**Comment by akumidv** [comment]
This will be implemented in new major version as settings inline parameters and parameters wo names.
